pages/index.py

```diff
@@ -61,6 +61,37 @@ def slider_item(title, text, active=""):
 <!--HOME SECTION START-->
 <section id="home">
 <div class="container">
+<div class="row">
+<!--DONATE SUBSECTION -->
+<div class="col-sm-12 col-md-12 col-lg-12">
+        <div class="item active donate" id="donate_button">
+             <p>$donate_text</p>
```

Comment:
Since `$donate_text` is generated using `md(…)`, the text here is already inside a `p`, hence the `<p>...</p>` here is a duplicate.